app/views/entries/new.html.erb

```diff
@@ -2,10 +2,8 @@
 
 <div class="section">
   <%= simple_form_for @entry do |f| %>
-  <div class="field">
-    <div class="control">
-      <%= f.input :list_id, input_html: { class: 'input' }, wrapper: false, label_html: { class: 'label' } %>
-    </div>
+  <%= f.association :list %>
```

Comment:
So I'm not that familiar with the association form helper but I wonder if you have to populate lists somehow for it to use?  I'm just curious as to how it works.